tensorflow/core/kernels/mkl_matmul_op.cc

```diff
@@ -0,0 +1,217 @@
+/* Copyright 2017 The TensorFlow Authors. All Rights Reserved.
```

Comment:
This file is part of a different pull request, which has been accepted already and is in the test phase.